File: c#/project/DAL/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL
{
    public class Comment
    {
        public int ID { get; set; }
        public string Content { get; set; }    
        public DateTime Date { get; set; }
        public int ProductId { get; set; }
        public virtual Product Product { get; set; }
        public int UserId { get; set; }
        public  virtual User User { get; set; }
    }
}

File: react-app/src/Componnents/features/comment/Comment.js

import DialogContent from '@mui/material/DialogContent';
import * as React from 'react'
const Comment = ({ comment }) => {
    return (
        <>
                <DialogContent>
                    {console.log(comment.id)}
                    <h3>{comment.date}</h3>
                    <h3>{comment.content}</h3>
                </DialogContent>
        </>
    )
}
export default Comment;

// public int ProductId { get; set; }
// public Product Product { get; set; }
// public int UserId { get; set; }
// public User User { get; set; }